.env.example

```
@@ -19,4 +19,10 @@ DATABASE=
 EMAIL_HOST=
 EMAIL_HOST_USER=
 EMAIL_HOST_PASSWORD=
-ADMIN_EMAIL=
\ No newline at end of file
+ADMIN_EMAIL=
+
+# AWS
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_DEFAULT_REGION=ap-northeast-2
+BACKUP_S3_BUCKET=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)